mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(taxPrice) {
+  let fullPrice = calculateSalesTax(taxPrice);
+  return "£" + fullPrice.toFixed(2)
```

Comment:
It is also possible to use string interpolation here =)
`return `£{fullPrice.toFixed(2)``